src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -105,6 +352,10 @@ function create_cache(mesh::VertexMappedMesh, equations,
     Pf = droptol!(sparse(Pf'), 100 * eps(eltype(Pf)))'
   end
 
+  # TODO: this is temporary, remove this once things are stable.
+  interp_matrix_gauss_to_face = TensorProductGaussFaceOperator(Interpolation(), dg)
+  Pf = TensorProductGaussFaceOperator(Projection(), dg)
```

Comment:
```suggestion
  # TODO: DGMulti; this is temporary, remove this once things are stable.
  interp_matrix_gauss_to_face = TensorProductGaussFaceOperator(Interpolation(), dg)
  Pf = TensorProductGaussFaceOperator(Projection(), dg)
```
What do you plan to use as a long term solution? Would it make sense to move this comment further up and comment out the construction of the other `Pf` and `interp_matrix_gauss_to_face`?

Reply:
Oops - this is another outdated comment. I'll remove it.